tests/html_vue/jsfmt.spec.js

```diff
@@ -1,5 +1,3 @@
 run_spec(__dirname, ["vue"]);
 run_spec(__dirname, ["vue"], { trailingComma: "es5" });
 run_spec(__dirname, ["vue"], { semi: false });
-run_spec(__dirname, ["vue"], { vueIndentScriptAndStyle: true });
```

Comment:
Why remove?
They are the only two test cases of `--vue-indent-script-and-style`.

Reply:
Feel free to add more tests in html_vue_indent if something's missing! I think adding `vueIndentScriptAndStyle` on the existing tests didn't provide any value.